config/action.d/iptables-allports.conf

```diff
@@ -42,6 +53,7 @@ actioncheck = <iptables> -n -L <chain> | grep -q 'f2b-<name>[ \t]'
 # Values:  CMD
 #
 actionban = <iptables> -I f2b-<name> 1 -s <ip> -j <blocktype>
+actionban6 = <ip6tables> -I f2b-<name>6 1 -s <ip> -j <blocktype6>
```

Comment:
as previous discussions pointed out -- it might be not effective to ban solely originating IPv6 ip alone but requires some reasonable (but what is reasonable?) netmask... what is your take on that?